client/layout/masterbar/item.jsx

```diff
@@ -51,12 +54,30 @@ export default React.createClass( {
 				onTouchStart={ this.preload }
 				onMouseEnter={ this.preload }>
 				{ !! this.props.icon &&
-					<Gridicon icon={ this.props.icon } size={ 24 } />
+				<Gridicon icon={ this.props.icon } size={ 24 } />
 				}
 				<span className="masterbar__item-content">{
-						this.props.children
+					this.props.children
 				}</span>
 			</a>
 		);
 	}
-} );
+}
+
+MasterbarItem.propTypes = {
+	url: React.PropTypes.string,
+	onClick: React.PropTypes.func,
+	tooltip: React.PropTypes.string,
+	icon: React.PropTypes.string,
+	className: React.PropTypes.string,
+	isActive: React.PropTypes.bool,
+	isNotesItem: React.PropTypes.bool,
+	preloadSection: React.PropTypes.func
+};
+
+MasterbarItem.defaultProps = {
```

Comment:
Both `propTypes` and `defaultProps` can also be declared as `static` properties on the `class`.